src/terms.jl

```diff
@@ -0,0 +1,296 @@
+import Base.+, Base.*, Base.length, Base.product
+
+# Abstract base class and definitions
+abstract Term
+
+function get_terms(term::Term)
+    return tuple(term)
+end
+
+function get_real_coefficients(term::Term)
+    return (Array(Float64, 0), Array(Float64, 0))
```

Comment:
I'm confused about what this function does.  It seems like it returns a tuple of zero-length arrays?